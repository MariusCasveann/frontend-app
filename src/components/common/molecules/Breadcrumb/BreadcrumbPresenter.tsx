import * as React from 'react';
import { Breadcrumb } from 'antd';

import './Breadcrumb.css';

export default () => (
    <div className="breadcrumb-padding breadcrumb-font">
        <Breadcrumb>
            <Breadcrumb.Item>Dashboard</Breadcrumb.Item>
            <Breadcrumb.Item>
                <a href="#/dashboard">Billing</a>
            </Breadcrumb.Item>
            <Breadcrumb.Item>Billing Item</Breadcrumb.Item>
        </Breadcrumb>
    </div>
);
