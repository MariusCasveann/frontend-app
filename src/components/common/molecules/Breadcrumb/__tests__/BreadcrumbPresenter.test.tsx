import * as React from 'react';
jest.doMock('antd', () => ({ 'Breadcrumb.Item': 'Breadcrumb.Item', Breadcrumb: 'Breadcrumb' }));

// components
import BreadcrumbPresenter from '../BreadcrumbPresenter';
import renderer from 'react-test-renderer';

describe('BreadcrumbPresenter', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<BreadcrumbPresenter />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
