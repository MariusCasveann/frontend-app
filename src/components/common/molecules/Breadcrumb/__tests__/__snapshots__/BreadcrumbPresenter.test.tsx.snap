// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadcrumbPresenter renders correctly 1`] = `
<div
  className="breadcrumb-padding breadcrumb-font"
>
  <div
    className="ant-breadcrumb"
  >
    <span>
      <span
        className="ant-breadcrumb-link"
      >
        Dashboard
      </span>
      <span
        className="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
    <span>
      <span
        className="ant-breadcrumb-link"
      >
        <a
          href="#/dashboard"
        >
          Billing
        </a>
      </span>
      <span
        className="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
    <span>
      <span
        className="ant-breadcrumb-link"
      >
        Billing Item
      </span>
      <span
        className="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
  </div>
</div>
`;
